src/main/scripts/org/reaktivity/specification/amqp/session/begin.exchange/client.rpt

```diff
@@ -0,0 +1,72 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect await ROUTED_SERVER
+        "nukleus://streams/amqp#0"
+  option nukleus:window 8192
+  option nukleus:transmission "duplex"
+
+connected
+
+# header exchange
+write "AMQP" [0x00 0x01 0x00 0x00]
+read "AMQP" [0x00 0x01 0x00 0x00]
+
+# open frame exchange
+write [0x00 0x00 0x00 0x18]                            # length = 24
+      [0x02]                                           # dof = 2
+      [0x00]                                           # type = AMQP (0)
+      [0x00 0x00]                                      # channel = 0
+      [0x00 0x53 0x10]                                 # performative = open (16)
+      [0xd0 0x00 0x00 0x00 0x08]                       # list type (list32), physical length = 8
+      [0x00 0x00 0x00 0x01]                            # number of fields in list = 1
+      [0xa1 0x02]                                      # type = string, length of string = 2
+      [0x63 0x31]                                      # container-id = c1
+
+read [0x00 0x00 0x00 0x19]                             # length = 25
+     [0x02]                                            # dof = 2
+     [0x00]                                            # type = AMQP (0)
+     [0x00 0x00]                                       # channel = 0
+     [0x00 0x53 0x10]                                  # performative = open (16)
+     [0xc0 0x0c]                                       # list type (list8), physical length = 12
+     [0x01]                                            # number of fields in list = 1
+     [0xa1 0x09]                                       # type = string, length of string = 9
+     [0x6c 0x6f 0x63 0x61 0x6c 0x68 0x6f 0x73 0x74]    # container-id = localhost
+
+#begin frame exchange
+write [0x00 0x00 0x00 0x1b]                            # length = 27
+      [0x02]                                           # dof = 2
+      [0x00]                                           # type = AMQP (0)
+      [0x00 0x00]                                      # channel = 0
+      [0x00 0x53 0x11]                                 # performative = begin (17)
+      [0xc0 0x0e]                                      # list type (list8), physical length = 14
+      [0x04]                                           # number of fields in list = 4
+      [0x40]                                           # type = null, remote-channel = null
+      [0x52 0x01]                                      # type = smalluint, next-outgoing-id = 1
+      [0x70 0x00 0x00 0x07 0xff]                       # type = uint, incoming-window = 2047
+      [0x70 0x7f 0xff 0xff 0xff]                       # type = uint, outgoing-window = 2147483647
```

Comment:
Suggest removing `type = ...` from comment, since each field in `begin` performative has a well-defined type.
Please make changes consistently across all similar files.

Reply:
Removed type information for all related files.